public/pages/Dashboard/Container/DashboardOverview.tsx

```diff
@@ -0,0 +1,263 @@
+import { AppState } from '../../../redux/reducers';
+import { CatIndex, IndexAlias } from '../../../../server/models/types';
+import {
+  getVisibleOptions,
+  isSystemIndices,
+} from '../../../pages/createDetector/containers/DataSource/utils/helpers';
+import {
+  DETECTOR_STATE,
+  PLUGIN_NAME,
+  APP_PATH,
+} from '../../../utils/constants';
+
+export function DashboardOverview() {
+  const dispatch = useDispatch();
+
+  const allDetectorList = useSelector(
+    (state: AppState) => state.ad.detectorList
+  );
+  // const isLoading = useSelector((state: AppState) => state.ad.requesting);
+
+  const [currentDetectors, setCurrentDetectors] = useState(
+    Object.values(allDetectorList)
+  );
+
+  const [allDetectorsSelected, setAllDetectorsSelected] = useState(true);
+
+  const [selectedDetectorsName, setSelectedDetectorsName] = useState(
+    [] as string[]
+  );
+  const getDetectorOptions = (detectorsIdMap: {
+    [key: string]: DetectorListItem;
+  }) => {
+    const detectorNames = Object.values(detectorsIdMap).map(
+      detectorListItem => {
+        return detectorListItem.name;
+      }
+    );
+    return detectorNames.map(buildItemOption);
+  };
+
+  const buildItemOption = (name: string) => {
+    return {
+      label: name,
+    };
+  };
+
+  const handleDetectorsFilterChange = (
+    options: EuiComboBoxOptionProps[]
+  ): void => {
+    const selectedNames = options.map(option => option.label);
+
+    setSelectedDetectorsName(selectedNames);
+    setAllDetectorsSelected(isEmpty(selectedNames));
+  };
+
+  const allDetectorStates = Object.values(DETECTOR_STATE);
+
+  const [selectedDetectorStates, setSelectedDetectorStates] = useState(
+    [] as string[]
+  );
+  // TODO: DetectorStates is placeholder for now until backend profile API is ready
+  const [allDetectorStatesSelected, setAllDetectorStatesSelected] = useState(
+    true
+  );
+
+  const getDetectorStateOptions = (states: string[]) => {
+    return states.map(state => {
+      return { label: state };
+    });
+  };
+
+  const handleDetectorStateFilterChange = (
+    options: EuiComboBoxOptionProps[]
+  ): void => {
+    const selectedStates = options.map(option => option.label);
+    setSelectedDetectorStates(selectedStates);
+    setAllDetectorStatesSelected(isEmpty(selectedStates));
+  };
+
+  const elasticsearchState = useSelector(
+    (state: AppState) => state.elasticsearch
+  );
+
+  const [selectedIndices, setSelectedIndices] = useState([] as string[]);
+  const [allIndicesSelected, setAllIndicesSelected] = useState(true);
+
+  const visibleIndices = get(elasticsearchState, 'indices', []) as CatIndex[];
+  const visibleAliases = get(elasticsearchState, 'aliases', []) as IndexAlias[];
+  const allIndices = [
+    ...visibleIndices
+      .filter(value => isSystemIndices(value.index))
+      .map(value => value.index),
+    ...visibleAliases
+      .filter(value => isSystemIndices(value.index))
+      .map(value => value.index),
+  ];
```

Comment:
use getVisibleOptions() helper fn here? see https://github.com/ohltyler/anomaly-detection-kibana-plugin/blob/updateDetectorListPage/public/pages/DetectorsList/List/List.tsx#L101

Reply:
it will return list of index options for combo box, which is not needed for filtering case here. I do use getVisibleOptions() in line 237 for combo box

Reply:
I see now, I did it a little different where I just ignore the filter altogether if the page state currently has all indices selected (in your case, allIndicesSelected set to true). This could maybe be a little more space efficient and eliminate some lines of code since you wouldn't need allIndices, but a small thing I guess.

Reply:
I see your. That is a good point. Will change to check if allIndicesSelected instead.